src/python/datastore/data_types.py

```diff
@@ -1326,6 +1331,12 @@ class FuzzerJob(Model):
   job = ndb.StringProperty()
   platform = ndb.StringProperty()
   weight = ndb.FloatProperty(default=1.0)
+  automated_weight = ndb.FloatProperty(default=1.0)
+
+  @property
+  def actual_weight(self):
+    """Get the actual weight for this job."""
+    return self.weight or self.automated_weight
```

Comment:
When will self.weight is false ? with default as 1.0, wont it always take precedence over automated_weight ?

Reply:
Should we return the product of these two? That way we'll always have equal CPU allocation by default, but special preference might be assigned through the manual weights.

Reply:
Should probably rename field if it is going to be a multiplier and not a simple weight.